packages/mui-material/src/styles/createPalette.js

```diff
@@ -58,7 +58,7 @@ export const dark = {
   },
   divider: 'rgba(255, 255, 255, 0.12)',
   background: {
-    paper: '#121212',
+    paper: '#1A2027',
```

Comment:
Would this mess with folks' layouts out there, if they're relying on Material as a default look? I'm sensing it will but wanted to get a confirmation.

Reply:
Components that have `backgroundColor: theme.palette.background.paper` will have the change applied.

If developers are not using their own custom background color, the change of this PR is supposed to improve the design.
If they are, the change of this PR is expected to not affect them.

So I think this PR is mergeable?

Reply:
Okay, it's what I thought. I'll iterate a bit more on this value so we're safe to merge.